src/server/server_family.cc

```diff
@@ -1478,7 +1480,10 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
     append("listpack_blobs", total.listpack_blob_cnt);
     append("listpack_bytes", total.listpack_bytes);
     append("small_string_bytes", m.small_string_bytes);
-
+    append("maxmemory", max_memory_limit);
+    append("maxmemory_human", HumanReadableNumBytes(max_memory_limit));
+    append("max_eviction_per_heartbeat", GetFlag(FLAGS_max_eviction_per_heartbeat));
```

Comment:
we usually do not expose flags like this. why do you need it?

Reply:
if user has a problem, I would need to get these values. Is there any other way to fulfill this?

Reply:
yes, we have multiple ways. 1. one can access `/flagz?` url of the server and see all the flags values.
2. `CONFIG GET *` provides an option to see all the values that are exposed for CONFIG. See `ConfigRegistry::Register` for details.

Reply:
great, thanks!

Reply:
> yes, we have multiple ways. 1. one can access `/flagz?` url on the server and see all the flags values. 2. `CONFIG GET *` provides an option to see all the values that are exposed for CONFIG. See `ConfigRegistry::Register` for details.

changed to register these two parameters via the `ConfigRegistry::Register`.